runtime/src/lib.rs

```diff
@@ -340,6 +340,11 @@ impl pallet_settlement_zksync::Config for Runtime {
     type WeightInfo = weights::pallet_settlement_zksync::NHWeight<Runtime>;
 }
 
+impl pallet_settlement_risc0::Config for Runtime {
+    type OnProofVerified = Poe;
+    type WeightInfo = weights::pallet_settlement_risc0::NHWeight<Runtime>;
```

Comment:
Maybe you neet to change it

Reply:
Yes, modified together with runtime tests.